### ISSUE TITLE
Which training set and test set did you use?

### ISSUE THREAD

**Issue by zihanchen0409** [kplabs-pl/ESA-ADB#8]
Hello, I was unable to reproduce your results. Which training set and test set did you use?

**Comment by Kotrix** [comment]
You should be able to reproduce the results using the instructions given in README file.

The exact data splits can be generated with https://github.com/kplabs-pl/ESA-ADB/blob/main/notebooks/data-prep/Mission1_semisupervised_prep_from_raw.py and the experiments can be reproduced with https://github.com/kplabs-pl/ESA-ADB/blob/main/mission1_experiments.py

Results for deterministic algorithms such as PCC, HBOS, STD, iForest, kNN should be exactly the same as reported in our paper. Let me know if this is not the case.

Results for deep learning algorithms such as Telemanom or DC-VAE may differ between machines as it is difficult to ensure full reproducibility due to stochastic nature of the algorithms. Let me know if you need the exact weights trained for our benchmark, we may consider sharing them.
